labgrid/driver/usbloader.py

```diff
@@ -181,10 +189,18 @@ def load(self, filename=None):
         mf = ManagedFile(filename, self.loader)
         mf.sync_to_resource()
 
-        cmd = ['-b', self.script] if self.script else []
+        # Convert usb path to the format the uuu-tool expects
+        # 1-1.2.1 -> 1:121
+        usb_device = str(self.loader.path).replace('.', '').replace('-', ':')
+
+        cmd = (
+            [self.tool, '-v', '-m', usb_device]
+            + ['-b', self.script] if self.script else []
```

Comment:
```suggestion
            + (['-b', self.script] if self.script else [])
```
I forgot to retest after refactoring the `cmd`. I'll force push the change